### ISSUE TITLE
Update Giving.cu.edu with 2021 Impact Report

### ISSUE THREAD

**Issue by Feissit** [CUCentralAdvancement/essential-cu#460]
We need to update the homepage of the giving site so that it links to the 2021 impact report

**Acceptance Criteria**

- [x] Banner design
- [x] Update giving site w/ links

![Screen Shot 2021-10-28 at 7 47 29 PM](https://user-images.githubusercontent.com/91230278/139359838-e6e414c6-e4b2-479a-8739-10630e924d54.png)

**Comment by duquennoisa** [comment]
@alexfinnarn is it possible to have the image swap out for mobile ? 

Its hard to make the image balance for all screen sizes
![image](https://user-images.githubusercontent.com/70037329/139729522-fc2f20ac-fd1c-4d94-8b30-a9991e95c3b6.png)

**Comment by alexfinnarn** [comment]
@duquennoisa It's currently not set up to do that, but it can be arranged. The current code is a bit of a hodgepodge, but it shouldn't be too hard to swap the background image...I would probably rearrange the template code but swapping the image should work.

How did you try to add this year's image to see how it looks? There are many different CSS background properties an even with a smaller/mobile image, some of those might need to change. It could also be the case that changing those CSS properties makes it possible to use one image.

**Comment by duquennoisa** [comment]
@alexfinnarn Last year I just worked it out in the design through trial and error so that it looks ok on all screens . If its complicated I can do the same this year too. It's just time consuming and it never ends up looking great. 

Let me know if you think we time/ability to try something else this year.

**Comment by alexfinnarn** [comment]
I'll only really know once I try it, so I can report back tomorrow on what I find out.

**Comment by alexfinnarn** [comment]
This can be done.

Sample PR that will need updating but lives in a different code repository: https://github.com/CUCentralAdvancement/cu-giving/pull/161

**Comment by duquennoisa** [comment]
@alexfinnarn Sound good. 
I created them to fit the size that is there now. It's basically centering and making the content bigger. 
Here are the files to update the banner: 

Desktop- https://www.dropbox.com/s/rxgtn49estrbwrx/IR21-givingsite-webBanner-alt-01.png?dl=0
Mobile -https://www.dropbox.com/s/blcb63v1hppqujf/IR21-givingsite-webBanner-mobile-01.png?dl=0

They should be live no earlier than 11/08

**Comment by alexfinnarn** [comment]
@duquennoisa I added this on https://giving-test.cu.edu/ with the breakpoint at 640px. At that width, the form moves under the image so I picked it for the breakpoint for the mobile image.

@imkleats @ktheiler The link on the banner includes UTM codes, so let me know what those should be. Currently, I have "https://essential.cu.edu/impact-reports/joy?utm_campaign=2021ImpactReport&utm_source=GivingWeb".
